spec/migrations/move_files_from_credential_to_folder.spec.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
```

Comment:
this file does not belong in this PR ! remove it ...
it was renamed on master !